site/docs/components/card/usage.mdx

```diff
@@ -8,31 +8,34 @@ data:
   $ref: ./#/data
 ---
 
-### Using the components
+### Using the component
+
+Cards may include text, images, or interactive elements. Make sure the content aligns with your use case, keeping it clear and easy to read. Avoid overcrowding cards with excessive content, actions, or links embedded in the text.
 
 #### When to use
 
 - To group related information with the same visual hierarchy.
 - To present a summary of a larger idea.
 - To link to more detailed information.
 
-#### When not to use
+#### When not to use
 
-- To display sequential information or emphasize ranking. Instead, use [`List`](../list).
-- As a stand-alone call to action. Instead, use [`Button`](../button).
+- To display sequential information or emphasize ranking. Instead, use [List](../list).
+- As a stand-alone call to action. Instead, use [Button](../button).
 - For decoration purposes only.
 
-### Card groups
+### Card groups
 
-- Group cards together in a layout using similar size and content length. If the content and size differs among cards, consider layout customization such as a Mosaic grid.
+- Group cards together in a layout using similar size and content length.
 - Avoid repeating images or content in a group of cards.
-- Use the same accent placement in a group if interactable.
+- It is recommended to use the same accent placement in a group.
 
-### Interactable elements
+### Interactable elements
+A default card can contain any UI element, including actionable components such as [Buttons](../button) or [Links](../link). You can use these, for example, to link to more detailed information.
 
-A default card can contain any UI elements, including actionable components such as [buttons](../button) or [links](../link). You can use these, for example, to link to more detailed information.
+Link cards shouldn't contain actionable components (like [Buttons](../button) or [Links](../link)), as the card itself is the interactive element.
 
-Interactable cards shouldn't contain actionable components (like [Buttons](../button) or [Links](../link)), as the card itself is the interactive element.
+Interactable cards serve as a [Toggle button](../toggle-button), [Button](../button), [Radio button](../radio-button) or radio group and [Checkbox](../checkbox) or checkbox group. They allow the user to embed these elements within for visual affordance.
```

Comment:
@mikearildbrown can you share guidance with the team on how components should be referenced:
- should we be using backticks here
- should we be using pascal case e.g. RadioButton